builds/misc/metrics-collector-images-release.yaml

```diff
@@ -4,108 +4,410 @@ pr: none
+    - job: MetricsCollector_Linux
+  ################################################################################
+      displayName: Metrics Collector Linux Build
+      timeoutInMinutes: 30
+      pool:
+        name: $(pool.linux.name)
+        demands:
+          - ImageOverride -equals agent-aziotedge-ubuntu-20.04-msmoby
+      steps:
+        # If you need to test this job. Create a new Service Connection in the Project Settings (Azure DevOps)
+        # and update the pipeline variable.
+        - task: Docker@2
+          displayName: Docker login edgerelease
+          inputs:
+            command: login
+            containerRegistry: $(service.connection)
+
+        - script: scripts/linux/buildBranch.sh -c $(Build.Configuration) --no-rocksdb-bin
+          name: build
+          displayName: Build ($(Build.Configuration))
+
+        - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+          displayName: 'SBOM Generation Task'
+          inputs:
+              BuildDropPath: '$(Build.BinariesDirectory)/publish'
+
+        - task: PublishBuildArtifacts@1
+          displayName: 'Publish Artifacts'
+          inputs:
+            PathtoPublish: '$(Build.BinariesDirectory)/publish'
+            ArtifactName: 'core-linux'
+
+        - task: DownloadPipelineArtifact@2
+          inputs:
+            artifact: 'librocksdb'
+            # place in '$(Pipeline.Workspace)/librocksdb'
+            path: '$(Pipeline.Workspace)/librocksdb'
+
+        - task: CopyFiles@2
+          displayName: Copy rocksdb libs
+          inputs:
+            sourceFolder: '$(Pipeline.Workspace)/librocksdb'
+            contents: '**'
+            targetFolder: '$(Build.BinariesDirectory)/publish/librocksdb'
+
+        - template: templates/image-linux.yaml
+          parameters:
+            name: Metrics Collector (customer-facing)
+            imageName: azureiotedge-metrics-collector
+            project: Microsoft.Azure.Devices.Edge.Azure.Monitor
+            version: $(version)
+        - template: templates/image-linux.yaml
+          parameters:
+            imageName: azureiotedge-agent
+            name: "Edge Agent"
+            project: Microsoft.Azure.Devices.Edge.Agent.Service
+            version: $(version)
+            buildx_flag: 'true'
+            use_rocksdb: true
+        - template: templates/image-linux.yaml
+          parameters:
+            imageName: azureiotedge-hub
+            name: "Edge Hub"
+            project: Microsoft.Azure.Devices.Edge.Hub.Service
+            version: $(version)
+            buildx_flag: 'true'
+            use_rocksdb: true
+        - template: templates/image-linux.yaml
+          parameters:
+            name: "Direct Method Sender"
+            imageName: azureiotedge-direct-method-sender
+            project: DirectMethodSender
+            bin_dir: '$(Build.BinariesDirectory)'
+            buildx_flag: 'true'
```

Comment:
Are these only here because you need them in the tests? Yuck (but ok). Can you add a comment?